rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/ast/TypeInterestFactory.java

```diff
@@ -26,277 +10,23 @@
  */
 public final class TypeInterestFactory
```

Comment:
TypeInterestFactory basically goes away. It may have to come back (temporarily) for this release, though.